### PR TITLE
Enable tracing

### PR DESCRIPTION
**Pull request by corps**
Allow us to track performance in sentry.io to investigate timeout issues.

### CODE REVIEW

File: src/seer/seer.py [hunk 1]
```diff
@@ -59,54 +59,48 @@ def model_path(subpath: str) -> str:
...
-            )
-
-        trends = {"data": [x[1] for x in trend_percentage_list]}
-        app.logger.debug("Trend results: %s", trends)
-
-        return trends
-    except Exception as e:
-        app.logger.exception("Error processing request")
-        return {"Error": str(e)}, 500
+    data = request.get_json()
```
Comment (corps) [inline src/seer/seer.py:77]: We shouldn't need the try/except wrapping, the `FlaskIntegration` from sentry handles this (and does so better).

File: src/seer/seer.py [hunk 2]
```diff
@@ -27,6 +26,7 @@ def traces_sampler(sampling_context):
     integrations=[FlaskIntegration()],
     traces_sampler=traces_sampler,
     profiles_sample_rate=1.0,
+    enable_tracing=True,
```
Comment (corps) [inline src/seer/seer.py:29]: This puts performance data of endpoints into sentry.io so we can watch capacity issues.

**Review by jennmueng — APPROVED** [review]
lgtm

**Comment by corps** [comment]
🙏

**Pull request merged**